Description: Change the default hinting level for subpixel rendering from "Full" to "Slight"
Author: ?

Index: gnome-control-center-2.31.90/capplets/appearance/appearance-font.c
===================================================================
--- gnome-control-center-2.31.90.orig/capplets/appearance/appearance-font.c	2010-08-17 02:46:46.000000000 +1000
+++ gnome-control-center-2.31.90/capplets/appearance/appearance-font.c	2010-09-01 17:43:05.662617001 +1000
@@ -963,7 +963,7 @@
                    ANTIALIAS_GRAYSCALE, HINT_FULL);
   setup_font_pair (appearance_capplet_get_widget (data, "subpixel_radio"),
                    appearance_capplet_get_widget (data, "subpixel_sample"),
-                   ANTIALIAS_RGBA, HINT_FULL);
+                   ANTIALIAS_RGBA, HINT_SLIGHT);
 
   font_render_load (data->client);
 
